packages/stems/src/components/HarmonyButton/BaseButton.module.css

```diff
@@ -20,7 +21,6 @@
 /* Only add hover styles on devices which support it */
 @media (hover: hover) {
   .button:not(.disabled):hover {
-    transition: all var(--hover);
```

Comment:
the transition property was only set while `:hover` so on hover end, we would perform the transition instantaneously